BridgeAppSDK/SBAUser.swift

```diff
@@ -43,8 +43,8 @@ public final class SBAUser: NSObject, SBAUserWrapper {
     
     let lockQueue = DispatchQueue(label: "org.sagebase.UserLockQueue", attributes: [])
 
-    public func logout() {
-        lockQueue.async {
+    public func resetStoredUserData() {
+        lockQueue.sync {
```

Comment:
Yes, but I do want this to happen before the code goes through setup and checks for what the current "user state" is *or* on logout. In both of those cases, the data needs to be cleared synchronously.